packages/grid/src/vaadin-grid-drag-and-drop-mixin.js

```diff
@@ -129,6 +129,24 @@ export const DragAndDropMixin = (superClass) =>
           e.stopPropagation();
         }
       });
+      this.__boundGridDragStartListener = this.__onGridDragStart.bind(this);
```

Comment:
If you do this binding in the constructor, you can drop the extra `addEventListener` (the function will be available on `connectedCallback`).

Also, no need to define a separate `bound...` function:

```js
constructor() {
  super();
  this.__onDocumentDragStart = this.__onDocumentDragStart.bind(this);
}
```

Reply:
Done.